README.md

```diff
@@ -12,43 +12,73 @@
  License.
 -->
 
-# Google Cloud Platform: github-actions
+# `setup-gcloud` GitHub Action
 
-This repository contains a library of [Github Actions](https://github.com/actions) providing functionality for working with [Google Cloud Platform](http://cloud.google.com/).
+Configures the [Google Cloud SDK][sdk] in the GitHub Actions environment. The Google Cloud SDK includes both the [gcloud][gcloud] and
+[gsutil][gsutil] binaries.
 
-## Available Actions
+Find other Google Cloud GitHub Actions in [`google-github-actions`](https://github.com/google-github-actions).
 
-* [setup-gcloud](./setup-gcloud/README.md): This action downloads, installs, and configures a [gcloud Cloud SDK](https://cloud.google.com/sdk/) environment for the worker, adding the `gcloud` CLI command to the worker's $PATH.
+**This repository also contains deprecated GCP GitHub Actions from previous mono-repo format.**
 
-* [get-secretmanager-secrets](./get-secretmanager-secrets/README.md): This action accesses secrets from [Google Secret Manager](https://cloud.google.com/secret-manager) and makes their results available as output variables.
+## Table of Contents
 
-## Example Workflows
+* [Usage](#usage)
+* [Inputs](#inputs)
+* [Example Workflows](#example-workflows)
+
+## Usage
+
+```yaml
+- name: Set up Cloud SDK
+  uses: google-github-actions/setup-gcloud@master
+  with:
+    version: '290.0.1'
```

Comment:
nit: I think we should remove the default version pinning, so that if someone is copying this code block they get latest by default.